test/bases.py

```diff
@@ -104,7 +123,7 @@ def _get_items_partial(self, item_pool, missing_item):
 
 
 class WorldTestBase(unittest.TestCase):
-    options: typing.Dict[str, typing.Any] = {}
+    options: typing.Dict[str, typing.Any] = DefaultOptionsDict()
```

Comment:
maybe this to more clearly illustrate the design intent?
```suggestion
    options: typing.ClassVar[typing.Dict[str, typing.Any]] = DefaultOptionsDict()
```

Reply:
This is intentional, because assigning options dynamically on instance level (in setUp) is an awkward but legit.

Reply:
Considering Zillion is the only game I've found currently merged that reassign options while a test is running (worlds/zillion/test/TestOptions.py), this would not break too much.